packages/js/src/plugins/auctionHouseModule/errors.ts

```diff
@@ -159,14 +160,20 @@ export class WithdrawFromBuyerAccountRequiresSignerError extends AuctionHouseErr
 
 /** @group Errors */
 export class PartialPriceMismatchError extends AuctionHouseError {
-  constructor(options?: MetaplexErrorOptions) {
+  constructor(
+    options: Partial<MetaplexErrorOptions> & {
+      expected: Amount;
+      actual: Amount;
+    }
+  ) {
```

Comment:
Sorry to be a pain but that's not how the other errors add arguments. Can we do it this way so it's more consistent?

```suggestion
  constructor(
    expected: Amount,
    actual: Amount,
    options?: MetaplexErrorOptions
  ) {
```